src/app/home/home.component.scss

```diff
@@ -0,0 +1,28 @@
+.row {
```

Comment:
Why you are modifying row class? Any specific reason? Also we already have grid and base file. Please import those to use row or col.